src/js/components/stock-movement/modals/RejectRequestModal.scss

```diff
@@ -0,0 +1,119 @@
+.reject-request-modal-content {
+  position: absolute;
+  top: 20%;
+  left: 23%;
+  background-color: #ffffff;
+  width: 55vw;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
this also looks like a copy of `configuration-modal-content` class so maybe we should define the default styling for modals somewhere globaly.